Makefile

```diff
@@ -131,3 +134,10 @@ vet: ## Apply go vet to all go files
 .PHONY: help
 help:
 	@grep -E '^[a-zA-Z/0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: revendor-test-e2e
+revendor-test-e2e: dep
+	curl --silent -o e2e_test.go https://raw.githubusercontent.com/openshift/cluster-api-actuator-pkg/master/pkg/e2e/e2e_test.go
```

Comment:
What is this???

Reply:
Takes the latest e2e_test.go so that dep has something to work on.